node-binance-api.js

```diff
@@ -1949,7 +1949,7 @@ let api = function Binance() {
 
                     symbols.forEach(symbolDepthInit);
                     let streams = symbols.map(function (symbol) {
-                        return symbol.toLowerCase() + '@depth';
+                        return symbol.toLowerCase() + '@depth' + '@depth@100ms';
```

Comment:
I think you should remove the '@depth' (also bellow)
just replace '@depth' with '@depth@100ms'

also just to make sure test it in a playground